@shared/api/internal.ts

```diff
@@ -279,7 +294,7 @@ export const retryAssetIcon = async ({
   } catch (e) {
     return assetIcons;
   }
-  const icon = await getIconUrlFromIssuer({ key, code });
+  const icon = await getIconUrlFromIssuer({ key, code, networkDetails });
```

Comment:
the only downside to not having a static network config is that in non-redux environments, we need to pass `networkDetails` as a param to any func that needs to access Horizon

Reply:
We could make the UI redux store a global, but it's a little messy to do so right now bc of monorepo structure. In the future, if passing around values becomes annoying, we can do some more work to do that